sys-libs/libcxx/libcxx-9999.ebuild

```diff
@@ -2,13 +2,14 @@
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI=5
+EAPI=6
 
-ESVN_REPO_URI="http://llvm.org/svn/llvm-project/libcxx/trunk"
+EGIT_REPO_URI="http://llvm.org/git/libcxx.git
+	https://github.com/llvm-mirror/libcxx.git"
 
-[ "${PV%9999}" != "${PV}" ] && SCM="subversion" || SCM=""
+[ "${PV%9999}" != "${PV}" ] && SCM="git-r3" || SCM=""
 
-inherit ${SCM} flag-o-matic toolchain-funcs multilib multilib-minimal
+inherit ${SCM} cmake-multilib toolchain-funcs
```

Comment:
nit: if you wish you could use multiple inherits instead of SCM variable; it seems commonly accepted these days